README.md

```diff
@@ -16,6 +16,16 @@ The following modifications to the original documents have been made:
   being added (see JSON files in [meta/speakers](meta/speakers/))
 - IDs and ID references reused in the DraCor documents have been sanitized
 
+## Corpus selection
+
+The tally of dramatic works in the Early Print corpus, as provided by its editors, amounted to 853 texts. 
+In this initial phase, we set aside the 363 texts that lack `<who-tags>` in the original markup. 
```

Comment:
I think "texts that lack speaker identification with `who` attributes" would technically be more precise.